src/charm.py

```diff
@@ -195,8 +194,28 @@ def _on_collect_unit_status(self, event: CollectStatusEvent):
 
         event.add_status(ActiveStatus())
 
+    def _generate_webui_config_file(self) -> str:
+        return render_config_file(
+            common_database_name=COMMON_DATABASE_NAME,
+            common_database_url=self._get_common_database_url(),
+            auth_database_name=AUTH_DATABASE_NAME,
+            auth_database_url=self._get_auth_database_url(),
+        )
+
+    def _configure_workload(self, restart: bool = False) -> None:
```

Comment:
Personally I'd write some docstring here explaining the conditions.

Reply:
A docstring is added to explain the conditions.